examples/multistage_trainer/data.py

```diff
@@ -0,0 +1,55 @@
+import tensorflow as tf
```

Comment:
can we use `edflow.datasets.mnist.MNIST` instead?

Reply:
fixed